sonobuoy-plugin.yaml

```diff
@@ -1,11 +1,7 @@
 sonobuoy-config:
   driver: Job
   plugin-name: k8s-service-validator-sonobuoy-plugin
-  result-format: raw
+  result-format: manual
 spec:
   image: yzaccc/k8s-service-validator:dev
   name: plugin
-  resources: {}
-  volumeMounts:
```

Comment:
These values aren't necessary; unless the version of sonobuoy is fairly old then it handles ensuring the right volume mounts for plugins